build/iptablesManager/iptablesManager-ds.yaml

```diff
@@ -0,0 +1,77 @@
+---
+kind: DaemonSet
+apiVersion: apps/v1
+metadata:
+  name: cloud-iptables-manager
+  namespace: kubeedge
+  labels:
+    app: iptables-manager
+spec:
+  selector:
+    matchLabels:
+      app: iptables-manager
+  template:
+    metadata:
+      labels:
+        app: iptables-manager
+    spec:
+      serviceAccount: iptables-manager-sa
+      hostNetwork: true
+      affinity:
+        nodeAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+            nodeSelectorTerms:
+            - matchExpressions:
+              - key: node-role.kubernetes.io/edge
+                operator: DoesNotExist
```

Comment:
Maybe we will have some cloud node, the iptablesmgr should also not deploy on them, so we need to selector the master node.

Reply:
> Maybe we will have some cloud node, the iptablesmgr should also not deploy on them, so we need to selector the master node.

OK, let's make it node affinity to the worker node.

Reply:
> > Maybe we will have some cloud node, the iptablesmgr should also not deploy on them, so we need to selector the master node.
> 
> OK, let's make it node affinity to the worker node.

And helm will support custom configuration if someone has this requirement.